src/main/java/com/xceptance/neodymium/util/SelenideAddons.java

```diff
@@ -211,47 +213,18 @@ public ElementsCollection get()
      */
     public static void $safe(final Runnable code)
     {
-        final int maxRetryCount = Neodymium.configuration().staleElementRetryCount();
-        int retryCounter = 0;
-
-        while (retryCounter <= maxRetryCount)
-        {
-            try
-            {
-                code.run();
-                break;
-            }
-            catch (final Throwable t)
-            {
-                if (isThrowableCausedBy(t, StaleElementReferenceException.class))
-                {
-                    retryCounter++;
-                    if (retryCounter > maxRetryCount)
-                    {
-                        // fail
-                        throw t;
-                    }
-                    else
-                    {
-                        AllureAddons.addToReport("StaleElementReferenceException catched times: \"" + retryCounter + "\".", retryCounter);
-                        Selenide.sleep(Neodymium.configuration().staleElementRetryTimeout());
-                    }
-                }
-                else
-                {
-                    // not the kind of error we are looking for
-                    throw t;
-                }
-            }
-        }
+        $safe(()->{
+            code.run();
+            return null;
+        });
     }
 
     private static boolean isThrowableCausedBy(final Throwable throwable, Class<? extends Throwable> clazz)
```

Comment:
I think we can change the access of this function to public. By doing so we are able to test it separately. 
Especially the nesting aspect should be covered.
What do you think?